components/Popup_bookmarked.js

```diff
@@ -22,9 +22,10 @@ function Popup(props) {
           <StyledDiv>Market Cap</StyledDiv>
           <StyledDiv>Volume (24h)</StyledDiv>
```

Comment:
Don't write directly inside a `<div>`. Use a `<p>`tag instead.

Please also control your code for further cases like this and fix them.

Reply:
done!